plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/fs/DBNPathBase.java

```diff
@@ -349,6 +360,15 @@ public void dropNodes(DBRProgressMonitor monitor, Collection<DBNNode> nodes) thr
         }
     }
 
+    private void copyInputStream(InputStream inputStream, Path folder, String str) throws IOException {
+        Path targetFile = folder.resolve(str);
+        CopyOption[] options = new CopyOption[0];
+        if (Files.exists(targetFile)) {
+            options = new CopyOption[] {StandardCopyOption.REPLACE_EXISTING};
+        }
+        Files.copy(inputStream, targetFile, options);
```

Comment:
Just make two distinct calls to` Files.copy`. The code will be cleaner. By the way, `StandardCopyOption.REPLACE_EXISTING` can be used when _creating_ new files. So you don't even to branch here.

```suggestion
        Files.copy(inputStream, targetFile, StandardCopyOption.REPLACE_EXISTING);
```